src/FffGcodeWriter.h

```diff
@@ -478,7 +478,22 @@ class FffGcodeWriter : public SettingsMessenger, NoCopy
      * \param infill_angle The angle in the XY plane at which the infill is generated.
      * \return Whether this function added anything to the layer plan
      */
-    bool processSkinPart(const SliceDataStorage& storage, LayerPlan& gcode_layer, const SliceMeshStorage* mesh, const int extruder_nr, const PathConfigStorage::MeshPathConfigs& mesh_config, const SkinPart& skin_part, unsigned int layer_nr, int skin_overlap, int infill_angle) const;
+    bool processSkinPart(const SliceDataStorage& storage, LayerPlan& gcode_layer, const SliceMeshStorage& mesh, const int extruder_nr, const PathConfigStorage::MeshPathConfigs& mesh_config, const SkinPart& skin_part, unsigned int layer_nr, int skin_overlap, int infill_angle) const;
+
+    /*!
+     * Add the g-code for sanding the top surface.
+     *
+     * This produces travel moves that cover the entire top surface.
```

Comment:
Indeed, there is a setting that insets the area. I've now edited the documentation to no longer say "entire", and in general made it a bit more clear: c4901204b85f6ec386b26904819e577b5df27525.